packages/wonder-blocks-dropdown/components/single-select.md

```diff
@@ -412,4 +412,31 @@ class ControlledSingleSelectExample extends React.Component {
 }
 
 <ControlledSingleSelectExample />
+```
+
+### Example: SingleSelect with custom opener
+
+In case you need to use a custom opener with the SingleSelect, you can use the `opener` property to achieve this:
+
+```js
+import {SingleSelect, OptionItem} from "@khanacademy/wonder-blocks-dropdown";
+import {View} from "@khanacademy/wonder-blocks-core";
+import IconButton from "@khanacademy/wonder-blocks-icon-button";
+import {icons} from "@khanacademy/wonder-blocks-icon";
+
+<SingleSelect
+    placeholder="Choose a juice"
+    opener={(eventState) => (
```

Comment:
What's the `eventState` used for? Could you provide an example of when/where that might be used?

Reply:
I'd like to encourage people to be using those states so that their custom openers can have a similar feel to the default openers.